### ISSUE TITLE
Exception while getting simple string from Web API

### ISSUE THREAD

**Issue by francotiveron** [fsprojects/SwaggerProvider#196]
### Description

Failure when calling a simple string returning GET call

### Repro steps

- create a F# web api app with the following controller

```fsharp
[<ApiController>]
type YatpApiController() = 
   inherit ControllerBase()

   [<HttpGet("version")>]
   member __.Get() = "0.0"
```

- create a F# client console application

```fsharp
module Yatp.Client

open SwaggerProvider
open System.Net.Http
open System

//let [<Literal>] Schema = "swagger.json"
let [<Literal>] Schema = @"http://localhost:5166/swagger/v1/swagger.json"
type YatpClient = OpenApiClientProvider<Schema>
let client = YatpClient.Client(new HttpClient(BaseAddress = Uri(@"http://localhost:5166/")))
let version = client.GetVersion().Result //Error
printfn "%A" version
```

Schema

```json
{
  "openapi": "3.0.1",
  "info": {
    "title": "Yatp.Api",
    "version": "1.0"
  },
  "paths": {
    "/version": {
      "get": {
        "tags": [
          "YatpApi"
        ],
        "responses": {
          "200": {
            "description": "Success",
            "content": {
              "text/plain": {
                "schema": {
                  "type": "string"
                }
              },
              "application/json": {
                "schema": {
                  "type": "string"
                }
              },
              "text/json": {
                "schema": {
                  "type": "string"
                }
              }
            }
          }
        }
      }
    },

```

### Expected behavior

the value "0.0" should be returned and then printed

### Actual behavior

An exception is raised in the client.GetVersion().Result line

`One or more errors occurred. (The JSON value could not be converted to System.String. Path: $ | LineNumber: 0 | BytePositionInLine: 3.)`

### Known workarounds

None

### Affected Type Providers

- [x] OpenApiClientProvider

### Related information

* Operating system: Windows 10
* .NET Runtime: .NET 6

NOTE:  C# client generated with NSwag or Visual Studio works as expected

```
var yatpClient = new Yatp.Client.YatpClient(@"http://localhost:5166/", new HttpClient());
var version = await yatpClient.VersionAsync(); //OK
Console.WriteLine(version);
```

**Comment by sergey-tihon** [comment]
Workaround here will be to modify server to return JSON `"0.0"` instead of plain text `0.0`

It can be done using `Produces(MediaTypes.ApplicationJson)` attribute

```fsharp
[<ApiController>]
type YatpApiController() = 
   inherit ControllerBase()

   [<HttpGet("version"); Produces(MediaTypes.ApplicationJson)>]
   member __.Get() = "0.0"
```

another option here is to inherit from provided client and override [member Deserialize](https://github.com/fsprojects/SwaggerProvider/blob/master/src/SwaggerProvider.Runtime/ProvidedApiClientBase.fs#L29), basically do nothing with response instead of default behavior `JsonSerializer.Deserialize(value, retTy, options)`

**Comment by sergey-tihon** [comment]
Fixed in [v2.0.0-beta3](https://www.nuget.org/packages/SwaggerProvider/2.0.0-beta3) by adding `Accept` header to requests, forcing server to respond with json.
